assets/js/components/product-preview/index.js

```diff
@@ -46,27 +46,29 @@ const ProductPreview = ( { product } ) => {
 
 	return (
 		<div
-			className="wc-product-preview"
+			className="wc-product-preview wc-block-grid__product"
```

Comment:
I'm not sure what you're asking– this is already the view component. Are you asking about reusing a component between PHP and JS?

Reply:
O you're right. One is PHP one is JS. My bad :) Not sure if there would be a way to share markup between them in that case..? Otherwise not an issue, we'll just document that changes need to be made in both placements.